### DISCUSSION TITLE
Working through SSH

### DISCUSSION THREAD

**Discussion by ntbol** [ThioJoe/YT-Spammer-Purge#784] [Q&A]
Trying to set this up on an Ubuntu server so I can SSH into it. I can get the script to run, but it doesnt auto open the browser Window for Google Auth. Is there a way around this?

**Comment by Firecul** [comment]
You could do the initial set-up on another device (not headless) and then transfer the token file to your server?

**Reply by ntbol** [reply]
Yeah that's a good idea. I'll give it a try, thanks! Will reply with results

**Reply by Firecul** [reply]
IIRC someone suggested an alteration to the process so it would display a link so you could do it like that but I don't recall what happened in the discussion. 
I just ssh into my PC so keep in mind you might have to reauth every now and then.

**Reply by ntbol** [reply]
Yeah the display link would be smart, ill look over the code and see if I can do that
